website/coursework/product-design/index.md

```diff
@@ -219,12 +224,16 @@ You should do some research before starting to work on the designs. Professional
 
 You might also find pattern libraries useful. A pattern library is a collection of user interface design elements, such as a login page, a news feed, a profile page, a credit card entry screen, etc. They are recurring solutions that solve common design problems.
 
-- [Mobbin](https://mobbin.design/) - A website created by fellow School of Computing student [Lim Ji Ho](http://www.jiholim.com/) that showcases the various screens found within popular mobile apps
-- [pttrns](https://pttrns.com/) - Another mobile app UI pattern library which includes Apple Watch apps
-- [Material Design Tools](https://material.io/tools/) - Customise and download Material Design icons and theme assets
-- [Apple Design Resources](https://developer.apple.com/design/resources/) - Assets and tools for designing for Apple devices
+- [Mobbin](https://mobbin.com/): A website that showcases the various screens found within popular mobile and web apps that's very popular among the design community. It is founded by School of Computing alumni [Jiho Lim](https://www.linkedin.com/in/jiho/) and [Liau Jian Jie](https://www.linkedin.com/in/liaujianjie/) (CS3216 AY2019/20).
```

Comment:
Update resources as some are no longer active or have move to a fully paid model.